amcl/src/amcl_node.cpp

```diff
@@ -308,10 +308,16 @@ class AmclNode
     // odom correction
     bool odom_correction_;
     bool is_init_odom_correction_;
+    bool pause_odom_correction_;
     pf_vector_t pure_amcl_delta_sum_;
     pf_vector_t pure_odom_delta_sum_;
     double odom_amcl_x_thre_;
     double odom_amcl_z_thre_;
+    double odom_sum_dist_;
```

Comment:
unused.

Reply:
deleted in 3df40539b9d00e3612ffeb3726cc5ee295daeee5